src/main/java/seedu/exercise/model/exercise/ValidationRegex.java

```diff
@@ -0,0 +1,25 @@
+package seedu.exercise.model.exercise;
+
+/**
+ * Encapsulates the different validation regex that help to check if an input is correct.
+ */
+public enum ValidationRegex {
```

Comment:
Should this class be in Util instead?

Reply:
Maybe it should be in commons package

Reply:
Should I rename it to ValidationRegexUtil and move it to the util package under commons?

Reply:
or just leave it as ValidationRegex and place it in the commons/core. So like Messages.java.

Reply:
Okay done.